src/site/content/en/fast/font-best-practices/index.md

```diff
@@ -189,90 +129,59 @@ serve stylesheets and fonts from separate origins. For example, this is how the
 
+### Using self-hosted fonts
 
-### Best practices
+On paper, using a self-hosted font should deliver better performance as it eliminates a third-party connection setup. However, in practice, the performance differences between these two options is less clear cut: for example, the [Web Almanac](https://almanac.httparchive.org/en/2020/fonts#fig-7) found that sites using third-party fonts had a faster render than fonts that used first-party fonts.
 
+If you are considering using self-hosted fonts, confirm that your site is using a [Content Delivery Network (CDN)](/content-delivery-networks/) and [HTTP/2](/content-delivery-networks/#http2-and-http3). Without use of these technologies, it is much less likely that self-hosted fonts will deliver better performance. For more information, see [Content Delivery Networks](/content-delivery-networks/).
 
-#### Using self-hosted fonts
+{% Aside %}
+    If you're unsure if using self-hosted fonts will deliver better performance, try serving a font file from your own servers and compare the transfer time (including connection setup) with that of a third-party font. If you have slow servers, don't use a CDN, or don't use HTTP/2 it becomes less likely that the self-hosted font will be more performant.
+{% endAside %}
 
-On paper, using a self-hosted font should deliver better performance as
-it eliminates a third-party connection setup. However, in practice, the
-performance differences between these two options is less clear cut: for
-example, the [Web Almanac](https://almanac.httparchive.org/en/2020/fonts#fig-7)
-found that sites using third-party fonts had a faster render than fonts that
-used first-party fonts.
+If you use a self-hosted font, it is recommended that you also apply some of the font file optimizations that third-party font providers typically provide automatically—for example, font subsetting and WOFF2 compression. The amount of effort required to apply these optimizations will depend somewhat on the languages that your site supports. In particular, be aware that optimizing fonts for [CJK languages](https://en.wikipedia.org/wiki/CJK_characters) ch
 
-If you are considering using self-hosted fonts, confirm that your site is using
-a [Content Delivery Network (CDN)](/content-delivery-networks/)
-and [HTTP/2](/content-delivery-networks/#http2-and-http3).
-Without use of these technologies, it is much less likely that self-hosted fonts
-will deliver better performance. For more information, see [Content Delivery
-Networks](/content-delivery-networks/).
+### Use WOFF2
 
-{% Aside %}
+Of the modern font fonts, [WOFF2](https://www.w3.org/TR/WOFF2/) is the newest, has the widest browser support, and offers the best compression. Because it uses Brotli, WOFF2 compresses 30% better than WOFF, leading to less data to download and therefore faster performance.
```

Comment:
I'd add that WOFF is entirely unnecessary if you don't need Internet Explorer support. WOFF2 alone is good enough most of the time!